newsplease/helper_classes/module_util.py

```diff
@@ -0,0 +1,20 @@
+import importlib
+
+
+class ClassLoader:
+	@classmethod
+	def from_string(cls, class_name):
+		if "." not in class_name:
+			raise ImportError("{0} does't look like a module path".format(class_name))
```

Comment:
1) please replace does't with doesn't
2) Why this test condition on containing "."? IIRC, is not only `some.path.modulename` a valid path, but for example only `modulename` is also a valid import path.

Reply:
Could you please rename the file to a more descriptive name, e.g., class_loader.py? Because currently it only contains the ClassLoader class and nothing else, thus we should go with the more descriptive name.

Reply:
@fhamborg About no2; the fully qualified name will be module.class at least. The use of a non names-paced class seemed rare enough I didn't see the value in handling that case. Do you disagree?

I'll do the other changes and update the PR soon. Apologies about the long delay.

Reply:
@fhamborg Changes are done.